CefSharp/IMenuModel.cs

```diff
@@ -4,8 +4,13 @@
 
 namespace CefSharp
 {
-	public interface IMenuModel
-	{
-		bool Clear();
-	}
+    public interface IMenuModel
+    {
+        int Count { get; }
```

Comment:
Do you mind adding `xml` documentation to these? Taking to improve documentation as a whole :+1: